src/scout_apm/celery.py

```diff
@@ -7,20 +7,21 @@
 from scout_apm.core.tracked_request import TrackedRequest
 
 
-# TODO: Capture queue.
-# https://stackoverflow.com/questions/22385297/how-to-get-the-queue-in-which-a-task-was-run-celery?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
-def prerun_callback(sender=None, headers=None, body=None, **kwargs):
-    name = kwargs["task"].name
+def prerun_callback(task=None, **kwargs):
```

Comment:
Changed method signature. Who calls this and from where?

Reply:
This is called by Celery itself, through its [signals](http://docs.celeryproject.org/en/v4.3.0/userguide/signals.html), as set up below in `install()`. It's always called with kwargs, so changing the signature here doesn't do anything for the caller, just inside the body of the function we only bind `task` and ignore the other parameters. According to [the docs for `task_prerun`](http://docs.celeryproject.org/en/v4.3.0/userguide/signals.html#task-prerun) the `headers` and `body` kwargs are not passed anyway, and `sender` is equal to `task`